skimage/measure/fit.py

```diff
@@ -39,6 +44,8 @@ class LineModel(BaseModel):
 
     A minimum number of 2 points is required to solve for the parameters.
 
+    **Deprecated, use ``LineModelND`` instead.**
```

Comment:
Is it what you meant @stefanv when saying "Deprecate LineModel over two releases" ?